quesma/quesma/schema_transformer.go

```diff
@@ -845,6 +845,41 @@ func (s *SchemaCheckPass) checkAggOverUnsupportedType(indexSchema schema.Schema,
 
 }
 
+func columnsToAliasedColumns(columns []model.Expr) []model.Expr {
+	aliasedColumns := make([]model.Expr, len(columns))
+	for i, column := range columns {
+		if columnRef, ok := column.(model.ColumnRef); ok {
+			aliasedColumns[i] = columnRef
+			continue
+		}
+		if col, ok := column.(model.LiteralExpr); ok {
+			if _, isStr := col.Value.(string); !isStr {
+				aliasedColumns[i] = model.NewAliasedExpr(column, fmt.Sprintf("column_%d", i))
+			} else {
+				aliasedColumns[i] = col
+			}
+			continue
+		}
+		if aliasedExpr, ok := column.(model.AliasedExpr); ok {
+			aliasedColumns[i] = aliasedExpr
+			continue
+		}
+		if _, ok := column.(model.FunctionExpr); ok {
+			aliasedColumns[i] = model.NewAliasedExpr(column, fmt.Sprintf("column_%d", i))
+			continue
+		}
+
+		aliasedColumns[i] = model.NewAliasedExpr(column, fmt.Sprintf("column_%d", i))
+		logger.Error().Msgf("Quesma internal error - unreachable code: unsupported column type %T", column)
```

Comment:
I would really love context here, though not sure if in this PR.

Reply:
Resolving as outside of scope of this PR.